polyphemus/lib/commands.rb

```diff
@@ -177,14 +177,66 @@ def add_string_attribute(attributes, name)
     end
   end
 
+  class CopyMetisFilesCommand < Etna::Command
```

Comment:
New command that copies files from one metis environment into another, filtered by a glob.  I'm using this to test against production flojo files in my development environment.  I intend to delete all data after finishing testing.  Still very useful for testing things like linking data locally without risking hurting production.